server/eclipse-project/src/main/java/us/freeandfair/corla/Main.java

```diff
@@ -223,6 +234,44 @@ private void activateEndpoints() {
   }
   
   /**
+   * Initializes the counties in the database using information from the 
+   * county properties.
+   */
+  private void initializeCounties() {
+    final Properties properties = new Properties();
+    try {
+      properties.load(ClassLoader.
+                      getSystemResourceAsStream(my_properties.getProperty(COUNTY_IDS)));
+    } catch (final IOException e) {
+      throw new IllegalStateException("Error loading county IDs, aborting.", e);
+    }
+    try {
+      for (final String s : properties.stringPropertyNames()) {
+        // if the property name is an integer, we assume it's a county ID
```

Comment:
I don't really think so. That file is the source of truth about county IDs; there isn't really anywhere to look for other validation, and I'm not aware (in Java) of any way to check if something is an integer aside from trying to parse it as an integer.